types/v3/tech-record/get/car/skeleton/index.d.ts

```diff
@@ -51,11 +51,11 @@ export interface TechRecordGETCarSkeleton {
   techRecord_applicantDetails_emailAddress?: null | string;
   createdTimestamp: string;
   partialVin?: null | string;
-  primaryVrm?: null | string;
+  primaryVrm: string;
```

Comment:
Should `skeleton.primaryVrm` be mandated? I would have expected the complete interface to mandate it? 🤔

Reply:
On a `GET` operation, `primaryVrm` should always be populated since if the user doesn't fill in the box on creation of the record, we generate a Z number from `cvs-svc-test-number` and stick it in the primaryVrm field. 


The idea of the `GET` object is to represent what the clients can expect back from the api, although since it's DynamoDB, one could argue that the only guaranteed fields are `systemNumber` and `createdTimestamp`. 

For all records generated from VTM the `primaryVrm` field will always be populated when doing a GET request. Only migrated records with data issues won't have a `primaryVrm` (unless they are trailers or course)